msteams-platform/toolkit/configure-bot-capability.md

```diff
@@ -0,0 +1,522 @@
+                       },
+                       "background": {
+                           "activeOnStart": true,
+                           "beginsPattern": ".*",
+                           "endsPattern": "listening to|Compiled|Failed|compiled|failed"
+                       }
+                   }
+               },
+                {
+                    "label": "Start application",
+                    "dependsOn": [
+                        "Start bot",
+                        "Start frontend"
+                    ]
+                },
+                {
+                    "label": "Start Teams App Locally",
+                    "dependsOn": [
+                        "Validate prerequisites",
+                        "Start local tunnel",
+                        "Provision",
+                        "Deploy",
+                        "Start application"
+                    ],
+                    "dependsOrder": "sequence"
+                },
+       ]
+   ```
+
+1. Add action `botAadApp/create` and `botFramework/create` under provision in `teamsapp.local.yml`.
+1. Add the following code in `file/createOrUpdateEnvironmentFile` action under deploy:
+
+   ```yml
+    provision:
+      - uses: botAadApp/create
+        with:
+          # The Microsoft Entra application's display name
+          name: bot-${{TEAMSFX_ENV}}
+        writeToEnvironmentFile:
+          # The Microsoft Entra application's client id created for bot.
+          botId: BOT_ID
+          # The Microsoft Entra application's client secret created for bot.
+          botPassword: SECRET_BOT_PASSWORD 
+    
+      # Create or update the bot registration on dev.botframework.com
+      - uses: botFramework/create
+        with:
+          botId: ${{BOT_ID}}
+          name: bot
+          messagingEndpoint: ${{BOT_ENDPOINT}}/api/messages
+          description: ""
+          channels:
+            - name: msteams
+    deploy:
+      - uses: file/createOrUpdateEnvironmentFile # Generate runtime environment variables
+        with:
+          target: ./tab/.localConfigs
+          envs:
+            BROWSER: none
+            HTTPS: true
+            PORT: 53000
+            SSL_CRT_FILE: ${{SSL_CRT_FILE}}
+            SSL_KEY_FILE: ${{SSL_KEY_FILE}}
+    
+      - uses: file/createOrUpdateEnvironmentFile # Generate runtime environment variables
+        with:
+          target: ./bot/.localConfigs
+          envs:
+            BOT_ID: ${{BOT_ID}}
+            BOT_PASSWORD: ${{SECRET_BOT_PASSWORD}}
+   ```
+
+   For more information, see the [sample project](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/hello-world-bot-with-tab).
+
+1. Open the **Run and Debug Activity Panel** and select **Debug (Edge)** or **Debug (Chrome)**.
+1. Select the F5 key to debug and preview your Teams app locally.
+
+## Provision the app to Azure
+
+1. Copy the `botRegistration/` folder from bot to `infra/`.
+1. Add following code to the bicep file:
+
+   ```json
```

Comment:
This JSON file isn't color coded. Generally, it is color coded for JSON scripts in review site.